plugins/Dashboard/lang/en.json

```diff
@@ -1,6 +1,7 @@
 {
     "Dashboard": {
         "AddAWidget": "Add a widget",
+        "AddWidgetToThisDashboard": "Add widget to this dashboard",
```

Comment:
Adding these new translation strings so they get translated by the community and in a month or so, when there are existing translations for most languages, we can update the UI to use them to be more descriptive as suggested in the original issue.